### PR TITLE
chore: Update Readme with project details, links, images

### PR DESCRIPTION
**Pull request by cafloyd**
_For an example of how to fill this template out, [see this Pull Request](https://github.com/the-collab-lab/tcl-3-smart-shopping-list/pull/44)._ 

## Description

Update Readme with description of the app, images, and links. Please feel free to edit my language/formatting and make additional changes/updates!

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|    | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|  ✓ | :scroll: Docs              |

### Before

image

### After

image

## Testing
See the updated readme [here](https://github.com/the-collab-lab/tcl-22-smart-shopping-list/tree/cf-update-readme)

### CODE REVIEW

**Review by kkatoch89 — APPROVED** [review]
I love this! So smart to include the video from the demo in it! Thanks @cafloyd!

**Comment by ablizben** [comment]
I like it! Thanks @cafloyd!

File: README.md [hunk 1]
```diff
@@ -1,146 +1,26 @@
...
-## What is this project?
+[AvoCart](https://tcl-22-shopping-list.web.app/) is a smart shopping list that learns your buying habits and helps you remember what you need to buy on your next trip to the store.
 
-The goal of this project is to collaborate with your Collab Lab team to create a “smart” shopping list app that learns your buying habits and helps you remember what you’re likely to need to buy on your next trip to the store.
+Developed by participants of [The Collab Lab](https://the-collab-lab.codes/)'s Spring 2021 cohort, using React.js, Firebase, Material UI, and Styled Components
 
-## How does it work?
+## Demo
 
-As a user, you will enter items (e.g., “Greek yogurt” or “Paper towels”) into your list. Each time you buy the item, you mark it as purchased in the list. Over time, the app comes to understand the intervals at which you buy different items. If an item is likely to be due to be bought soon, it rises to the top of the shopping list.
```
Comment (dariatsvetkova) [inline README.md:9]: I like the use of video here. I think we should also keep the text description as an option to learn about the project without watching the video.

**Review by dariatsvetkova — APPROVED** [review]
It's a great idea! Thanks, @cafloyd!

File: README.md (hunk 1 above)
Comment (cafloyd) [reply README.md:9]: Great idea! Added the "How does it work?" section back to the readme.

**Review by ablizben — APPROVED** [review]
Did we want to add a link to Figma on here?

**Review by luisaugusto — APPROVED** [review]

**Pull request merged**